pkgs/applications/window-managers/yabar/default.nix

```diff
@@ -1,38 +1,49 @@
-{ stdenv, fetchFromGitHub, cairo, gdk_pixbuf, libconfig, pango, pkgconfig, xcbutilwm }:
+{ stdenv, fetchFromGitHub, cairo, gdk_pixbuf, libconfig, pango, pkgconfig
+, xcbutilwm, alsaLib, wirelesstools, asciidoc, libxslt, makeWrapper
+, configFile ? null, lib
+}:
 
 stdenv.mkDerivation rec {
   name    = "yabar-${version}";
-  version = "0.4.0";
+  version = "2017-09-09";
```

Comment:
I'd suggest introducing a package `yabar-unstable` that tracks the git version and leave the `yabar` at the released version.

Reply:
(at least if upstream plan to make future stable releases)

Reply:
There is an open discussion about it (https://github.com/geommer/yabar/issues/136)

As there's no activity, I think it's indeed the better approach to add an `unstable.nix`.